com.unity.render-pipelines.core/Runtime/Debugging/VolumeDebugSettings.cs

```diff
@@ -25,6 +25,24 @@ public Camera selectedCamera
             set => m_SelectedCamera = value;
         }
 
+        /// <summary>Selected camera index.</summary>
+        public int selectedCameraIndex
+        {
+            get
+            {
+                var index = -1;
+                var c = cameras.ToArray();
+                if (m_SelectedCamera != null && c.Any())
+                    index = Array.IndexOf(cameras.ToArray(), m_SelectedCamera);
```

Comment:
not API related...
can't you reuse `c` instead of of calling `camera.ToArray()` again ?

Reply:
good catch, done